image/tools/lib/component/codeready_pv.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+function dump_pod_data {
+    workspace_pod_name=$1
+    dump_dest=$2
+    workspace_id=$(echo ${workspace_pod_name} | awk -F"." '{ print $1}')
+    cp_pod_data "codeready/${workspace_pod_name}:/projects" "${dump_dest}/${workspace_id}"
+}
+
+function component_dump_data {
+    workspace_pods=$(oc get pods -n codeready | grep workspace | awk '{print $1}')
+
+    if [ ${#workspace_pods} = 0 ]
+    then
+        echo "No workspaces found to backup"
+    else
+        archive_path=$1
```

Comment:
You have to add `/archives` to the path, because that's what the signing part expects:
https://github.com/integr8ly/backup-container-image/blob/master/image/tools/entrypoint.sh#L39

IMHO it should be a part of passed param. We can fix/improve that later, because it will require update across all component scripts.

Reply:
So you mean `"$1/archives/"`?

Reply:
ah nvm, I see it.

Reply:
With current code I am getting this error:

```
gpg: can't open `/tmp/intly/archives/*': No such file or directory
gpg: /tmp/intly/archives/*: encryption failed: No such file or directory
==> Data encryption completed
ERROR: Parameter problem: Nothing to upload.
==> Upload ==>: FAILED
```

Reply:
Thanks @matskiv. Updated now.